data/json/furniture_and_terrain/furniture-plumbing.json

```diff
@@ -129,7 +129,42 @@
         { "item": "hose", "count": [ 0, 2 ] },
         { "item": "cu_pipe", "count": [ 1, 4 ] },
         { "item": "scrap_copper", "count": [ 0, 2 ] },
-        { "item": "water_faucet", "count": [ 0, 1 ] }
+        { "item": "water_faucet", "count": [ 0, 1 ] },
+        { "item": "material_aluminium_ingot", "count": 2 }
```

Comment:
why a whole ingot? i doubt water heater should drop a big solid blocks of aluminum, more like a lot of aluminum scrap

Reply:
The name is misleading; these “ingots” are 250g each.

Reply:
"ingot" is not about the shape, it's about the form of an item

Reply:
Hmm. Well, the anode rod is in fact a rod longer than it is wide. Usually masses around a kilo, and is often divided into four segments…

Reply:
You can do a google image search to see pictures if you want.

Reply:
ok, then it require some tweaks for aluminum to have more components, but it's out of scope of this PR